transport/src/main/java/io/netty/channel/IoExecutionContext.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2024 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel;
+
+/**
+ * The execution context for an {@link IoHandler}.
+ * All method must be called from the {@link EventLoop} thread.
```

Comment:
```suggestion
 * All methods must be called from the {@link EventLoop} thread.
```

Reply:
The same typo exits on main too

Reply:
I will "forward port" all changes that make sense to main